contrib/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+pytest-env
```

Comment:
imo we could just require the `testing` extra here instead of adding another `requirements.txt` file.